configs/vision/owkin/phikon/offline/panda.yaml

```diff
@@ -0,0 +1,129 @@
+---
+trainer:
+  class_path: eva.Trainer
+  init_args:
+    n_runs: &N_RUNS ${oc.env:N_RUNS, 5}
+    default_root_dir: &OUTPUT_ROOT ${oc.env:OUTPUT_ROOT, logs/owkin/phikon/offline/panda}
```

Comment:
same here as above; `default_root_dir: &OUTPUT_ROOT ${oc.env:OUTPUT_ROOT, logs}/owkin/phikon/offline/panda`

Reply:
(same solution as suggested in comment above)